### PR TITLE
Add Salesforce to adopter list

### PR DESCRIPTION
**Pull request by b-hoyt**

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ URLs if possible) and [submit a pull request][1]!
   * [Thor, a Finagle-based image processor][4]
 * [Reputation.com](https://www.reputation.com/)
 * [Sabre Labs](http://sabrelabs.com/)
+* [Salesforce](http://www.salesforce.com/)
```
Comment (mosesn) [inline ADOPTERS.md]: do you mind if we use the https url instead?

Comment (b-hoyt) [inline ADOPTERS.md]: Fixed

**Comment by b-hoyt** [comment]
Primarily thrift servers and clients, http clients, dtabs. Trending towards linkerd + grpc to accommodate our polyglot environment.

**Comment by vkostyukov** [comment]
Thanks a lot @b-hoyt! I'm pulling this internally. Will close this PR once it's merged in.

**Comment by vkostyukov** [comment]
This has been merged internally and is already [synced with Github](https://github.com/twitter/finagle/commit/f4b23dc18a2dcd30303363e045f15979667e35e2). Thanks again @b-hoyt!

**Comment by b-hoyt** [comment]
Thanks @vkostyukov @mosesn and everyone else who contributes to Finagle! The best practices, patterns, and hard-earned experience encoded here are invaluable to anyone aspiring to operate a services architecture. Very fortunate to benefit from this project and look forward to contributing back.

**Pull request closed without merging**